pages/docs/installation/installation-packages.md

```diff
@@ -12,18 +12,18 @@ This section lists systems and instructions on how to install these packages.
 You can download version-specific Ubuntu (Debian) packages from each [GitHub release](https://github.com/precice/precice/releases/latest).
 To install, simply open it in your software center.
 
-Alternatively, download & install it from the command line. For **Ubuntu 20.04 (focal)**:
+Alternatively, download & install it from the command line. For **Ubuntu 22.04 (jammy)**:
 
 ```bash
-wget https://github.com/precice/precice/releases/download/v{{ site.precice_version }}/libprecice2_{{ site.precice_version }}_focal.deb
-sudo apt install ./libprecice2_{{ site.precice_version }}_focal.deb
+wget https://github.com/precice/precice/releases/download/v{{ site.precice_version }}/libprecice2_{{ site.precice_version }}_jammy.deb
+sudo apt install ./libprecice2_{{ site.precice_version }}_jammy.deb
 ```
```

Comment:
Same comment as above with variables.

Reply:
Let's not implement this with this PR and reconsider it if updating becomes a burden or if we continuously run into inconsistencies.